clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -191,7 +191,7 @@ public class TransactionManager {
     private volatile RuntimeException lastError = null;
     private volatile ProducerIdAndEpoch producerIdAndEpoch;
     private volatile boolean transactionStarted = false;
-    private volatile boolean epochBumpRequired = false;
+    private volatile boolean epochBumpTriggerRequired = false;
```

Comment:
Are we using the word "trigger" to refer to the client-initiated epoch bumps? I'm wondering if this is just the client side ones, if we can be specific here.

Something like "clientSideEpochBumpRequired"

Reply:
Yes I was going to use that but decided on trigger in the end, if this naming seems better to you I can change it!